.github/scripts/assert.sh

```diff
@@ -1,248 +0,0 @@
-#!/usr/bin/env bash
-
```

Comment:
was used to have assertions in bash. Not needed now since:

1. ButtonDeath event carries no data anyway (it would be cumbersome to return it as Button::death and IButton::death would have different signatures, plus there is no need since this data can be queried) 
2. parity merged a PR that factors out transcode bits form cargo-contract into a standalone library, so next PR or next-next PR can introduce proper (Rust) e2e test